pg2mysql.pl

```diff
@@ -136,6 +136,10 @@ sub handle_create {
     
     debug_print("input line is $line\n");
 
+    if ( $line =~ m/\s*CONSTRAINT .*? CHECK/ ) {
+        $line = handle_check($line);
+    }
```

Comment:
Should this be an else? Or do you actually want to go through the rest of the line conversion below?

Reply:
I guess you could also just return as well